types/redux-api-middleware/index.d.ts

```diff
@@ -13,13 +13,14 @@ import {
     applyMiddleware
 } from 'redux';
 
+type TypeOrResolver<Arg, Type> = Type | ((arg: Arg) => Type);
+
 /**
  * This module is also a UMD module that exposes a global variable 'ReduxApiMiddleware'
  * when loaded outside a module loader environment.
  */
 export as namespace ReduxApiMiddleware;
 
-export type RSAA = '@@redux-api-middleware/RSAA';
```

Comment:
Why eliminate the type?

Reply:
It wasn't used. `redux-api-middleware` exports just this const. And `RSAAAction` uses `RSAA` as a value not a type.
This happens when only type exists.
```
'RSAA' only refers to a type, but is being used as a value here.
```